Makefile

```diff
@@ -182,3 +182,14 @@ tools: # install dependencies and tools required to build
 	$(GO_CMD) generate -tags tools tools/tools.go
 	@echo
 	@echo "Done!"
+
+.PHONY: test/boltdbstate
+test/boltdbstate:
+	@echo "Running state tests..."
+	go test -test.v ./internal/server/boltdbstate 
+
+
+.PHONY: test/service
+test/service:
+	@echo "Running service API server tests..."
```

Comment:
Maybe add a comment here that mentions users should run `docker-compose up` first to run this? I wouldn't necessarily expect this target to do that for users, but we should mention it if this these tests will fail otherwise.

Reply:
Good idea!

Reply:
Pushed that up 👍🏻